Common/include/geometry/dual_grid/CPoint.hpp

```diff
@@ -771,14 +771,14 @@ class CPoint {
   }
 
   /*!
-   * \brief Get the value of the old coordinates for implicit smoothing.
+   * \brief Get the value of the old coordinates for implicit smoothing. <- Nijso: really?
    * \param[in] iPoint - Index of the point.
    * \return Old coordinates at a point.
    */
   inline su2double* GetCoord_Old(unsigned long iPoint) { return Coord_Old[iPoint]; }
```

Comment:
I do not think this is used for implicit smoothing, but is necessary for unsteady restarts?

Reply:
Something like that, or finite difference grid velocities. You can remove your notes and merge.